rotkehlchen/types.py

```diff
@@ -523,3 +510,16 @@ def deserialize(cls: Type['AddressbookEntry'], data: Dict[str, Any]) -> 'Address
 class AddressbookType(SerializableEnumMixin):
     GLOBAL = 1
     PRIVATE = 2
+
+
+class DataImportSource(SerializableEnumMixin):
+    COINTRACKING = 1
+    CRYPTOCOM = 2
+    BLOCKFI_TRANSACTIONS = 3
+    BLOCKFI_TRADES = 4
+    NEXO = 5
+    GITCOIN = 6
+    SHAPESHIFT_TRADES = 7
+    UPHOLD_TRANSACTIONS = 8
+    BISQ_TRADES = 9
+    BINANCE = 10
```

Comment:
I did not add a new `serialize` method since this enum is not supposed to be used as a json key in a response. It is supposed to only transform incoming csv format from the frontend (and `deserialize` supports snake case, so no changes needed imo).

Reply:
yes fair enough if it 's not needed it's fine.